src/arrayMethodSort.js

```diff
@@ -5,8 +5,31 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    if (!compareFunction) {
+      for (let i = 0; i < this.length - 1; i++) {
+        for (let j = 0; j < this.length - i - 1; j++) {
+          if (String(this[j]) > String(this[j + 1])) {
```

Comment:
You should avoid unnecessary type conversion and explicitly compare the items. If you're sorting numbers, you could use a numerical comparison instead of a string comparison to avoid unexpected results.